feathr_project/feathr/registry/_feature_registry_purview.py

```diff
@@ -93,6 +93,14 @@ def _register_feathr_feature_types(self):
 
                 AtlasAttributeDef(
                     name="path", typeName="string", cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(
+                    name="url", typeName="string", cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(
+                    name="dbtable", typeName="string", cardinality=Cardinality.SINGLE),
```

Comment:
Can we add those in a customized dict or something so it's more flexible, rather than adding it here in the attributes? Reasons being:
1. Those schemas are hard (or impossible) to change
2. Not everyone is using JDBC
Putting those in a `map<string,string>` allows us to have more flexibility.

Reply:
I don't plan to do larger refactor to this part in this PR, yihui is working on the data model re-design and will touch this as well. So let keep this change as simple as possible, to reduce the probability to have conflict with her change.

BTW all these fields are optional so it won't break existing HDFS support.

Reply:
> 

As to `--system-properties`, if you look into the code on Scala side, you'll find that the name does serve the purpose.